knimin/handlers/barcode_util.py

```diff
@@ -201,6 +201,9 @@ def get(self):
         project_names = db.getProjectNames()
 
         # barcode exists get general info
+        # TODO (Stefan Janssen): check spelling of "received", i.e. tests in
```

Comment:
In the DB, it is written `"Received"`, but in the template it is used like
`{% for bstatus in ['Recieved', ''] %}
    {% if bstatus == barcode_info['status'] %}`
So I don't see how that condition can ever be satisfied.